### PR TITLE
CSS coding standard fixes

### PR DESCRIPTION
**Pull request by sunpietro**
| Question      | Answer
| ------------- | ---
| Tickets       | N/A
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

## For QA

You have to test the look of:
- images in the image field type,
- images in the image asset field type,
- button icons in AlloyEditor UI

### CODE REVIEW

File: src/bundle/Resources/public/js/alloyeditor/src/base/ez-blocktextalign.js [hunk 1]
```diff
@@ -59,10 +59,9 @@ export default class EzBlockTextAlign extends Component {
         const icon = '/bundles/ezplatformadminui/img/ez-icons.svg#' + this.props.iconName;
 
         return (
-            <button className={cssClass} onClick={this.applyStyle.bind(this)}
-                tabIndex={this.props.tabIndex} title={this.props.label}>
-                <svg className='ez-icon ez-btn-ae__icon'>
-                    <use xlinkHref={icon}></use>
+            <button className={cssClass} onClick={this.applyStyle.bind(this)} tabIndex={this.props.tabIndex} title={this.props.label}>
+                <svg className="ez-icon ez-icon--small ez-icon--light ez-btn-ae__icon">
```
Comment (dew326) [inline src/bundle/Resources/public/js/alloyeditor/src/base/ez-blocktextalign.js]: I think there is no point to add the `ez-icon-light` class. We are about to change the styling of AE and we will have to remove this class.

**Comment by sunpietro** [comment]
@lserwatka branch has just been rebased.
ping @dew326 for review

**Comment by lserwatka** [comment]
@katarzynazawada have you started testing here?

**Comment by katarzynazawada** [comment]
@lserwatka not yet, I will start

**Pull request merged**